github4s/jvm/src/test/scala/github4s/unit/ApiSpec.scala

```diff
@@ -917,6 +917,37 @@ class ApiSpec
     }
   }
 
+  "Issues >> ListComments" should "return the expected issue comments when a valid issue number is provided" in {
+    val response =
+      issues.listComments(
+        accessToken,
+        headerUserAgent,
+        validRepoOwner,
+        validRepoName,
+        validIssueNumber)
+    response should be('right)
+
+    response.toOption map { r ⇒
+      r.result.nonEmpty shouldBe true
+      r.statusCode shouldBe okStatusCode
+    }
+  }
+  it should "return an error if an invalid issue number is provided" in {
+    val response =
+      issues.listComments(
+        accessToken,
+        headerUserAgent,
+        validRepoOwner,
+        validRepoName,
+        invalidIssueNumber)
+    response should be('left)
+  }
+  it should "return an error if no tokens are provided" in {
+    val response =
+      issues.listComments(None, headerUserAgent, validRepoOwner, validRepoName, validIssueNumber)
+    response should be('left)
+  }
+
```

Comment:
I think you're missing the mocked responses: http://47deg.github.io/github4s/contributing.html#mocking-the-responses

Reply:
also true